injector/src/test/java/edu/ucr/cs/riple/injector/ComprehensiveTest.java

```diff
@@ -319,44 +319,27 @@ public void methodWithMultipleLineDeclaration() {
         .addChanges(
             new RemoveAnnotation(
                 new OnMethod(
-                    "Main.java",
-                    "edu.ucr.Main",
-                    "foo6(Map<String, Object>, Object, Object, Object, Object)"),
+                    "Main.java", "edu.ucr.Main", "foo6(Map, Object, Object, Object, Object)"),
```

Comment:
Why do we just see `Map` and `Object` here rather than `java.util.Map` and `java.lang.Object`?

Reply:
@msridhar These tests are added through the merging of Injector to Annotator. Previously Injector was written as a separate standalone project and some of the tests cases are from previous repository. Injector can work with both full name and simple names if fully qualified names are not given as input.


Although annotator-core always gives injector fully qualified names, some of the tests are written with simple names as it was designed regardless of getting used only in annotator-core. Please let me know if you would like me to change this as Injector is no longer a standalone project. Thank you.